media-tv/plex-media-server/plex-media-server-1.22.2.4282.ebuild

```diff
@@ -0,0 +1,98 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit systemd unpacker
+
+DESCRIPTION="A free media library that is intended for use with a plex client"
+HOMEPAGE="https://www.plex.tv/"
+
+MY_COMMIT="a97b03fad"
+MY_PV="${PV}-${_COMMIT}"
+
+MY_URI="https://downloads.plex.tv/plex-media-server-new"
+#URI="https://artifacts.plex.tv/plex-media-server-experimental/"
+SRC_URI="
+	amd64? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_amd64.deb )
+	x86? ( ${MY_URI}/${MY_PV}/debian/plexmediaserver_${MY_PV}_i386.deb )"
+
+LICENSE="Plex"
+SLOT="0"
+KEYWORDS="-* ~amd64 ~x86"
+RESTRICT="mirror bindist strip"
+IUSE="system-openssl"
+
+BDEPEND="dev-util/patchelf"
+
+RDEPEND="
+	acct-group/plex
+	acct-user/plex
+	system-openssl? ( dev-libs/openssl )
+"
+
+QA_PREBUILT="*"
+QA_MULTILIB_PATHS=(
+    "usr/lib/plexmediaserver/lib/.*"
+    "usr/lib/plexmediaserver/Resources/Python/lib/python2.7/.*"
+    "usr/lib/plexmediaserver/Resources/Python/lib/python2.7/lib-dynload/_hashlib.so"
+)
+	"usr/lib/plexmediaserver/Resources/Python/lib/python2.7/.*"
+	"usr/lib/plexmediaserver/Resources/Python/lib/python2.7/lib-dynload/_hashlib.so" )
+
+PATCHES=(
+    "${FILESDIR}/${PN}-1.22.2.4282-desktop-file.patch"
+)
+
+S="${WORKDIR}"
+
+src_prepare() {
+	eapply_user
+}
+
+src_install() {
+	# Remove Debian specific files
+	rm -r "${S}/usr/share/doc" || die
+
+	# Remove shipped openssl library
+	if use system-openssl; then
+		rm -f usr/lib/plexmediaserver/libssl.so.1.0.0 || die
+	fi
+
+	# Add startup wrapper
+	dosbin "${FILESDIR}/start_pms" || die
+
+	# Copy main files over to image and preserve permissions so it is portable
+	cp -rp usr/ "${ED}" || die
+
+	# Make sure the logging directory is created
+	keepdir /var/log/pms
+	fowners plex:plex /var/log/pms
+
+	keepdir /var/lib/plexmediaserver
+	fowners plex:plex /var/lib/plexmediaserver
+
+	doinitd "${FILESDIR}"/init.d/${PN}
+	doconfd "${FILESDIR}"/conf.d/${PN}
+
+	systemd_dounit ${ED}/usr/lib/plexmediaserver/lib/plexmediaserver.service
+	keepdir /var/lib/plexmediaserver
+
+	# Adds the precompiled plex libraries to the revdep-rebuild's mask list
+	# so it doesn't try to rebuild libraries that can't be rebuilt.
+	dodir /etc/revdep-rebuild
+	insinto /etc/revdep-rebuild
+	doins "${FILESDIR}"/etc/revdep-rebuild/80plexmediaserver
+
+	# Fix RPATH
+	patchelf --force-rpath --set-rpath '$ORIGIN:$ORIGIN/../../../../../../lib' "${ED}"/usr/lib/plexmediaserver/Resources/Python/lib/python2.7/lib-dynload/_codecs_kr.so || die
+	patchelf --force-rpath --set-rpath '$ORIGIN:$ORIGIN/../../../../../../lib' "${ED}"/usr/lib/plexmediaserver/Resources/Python/lib/python2.7/lib-dynload/_multibytecodec.so || die
+
+}
+
+pkg_postinst() {
+	einfo "Plex Media Server is now installed. Please check the configuration file 
```

Comment:
I bet we want to either make multiple `einfo` calls here _or_, possibly even better, use [readme.gentoo-r1](https://devmanual.gentoo.org/eclass-reference/readme.gentoo-r1.eclass/index.html)?

Reply:
This should be in good shape, now.